docs/advanced-guide/injecting-databases-drivers/page.md

```diff
@@ -104,6 +111,13 @@ type Mongo interface {
 
 User's can easily inject a driver that supports this interface, this provides usability without
 compromising the extensibility to use multiple databases.
+
+Import the gofr's external driver for MongoDB:
+
+```go
+go get gofr.dev/pkg/gofr/datasource/mongo
+```
```

Comment:
The fenced block type is wring (here and all `go get` you add)


````suggestion
```shell
go get gofr.dev/pkg/gofr/datasource/mongo
```
````

Reply:
Sure done